metrics/prometheus-agent/values.yaml

```diff
@@ -32,8 +34,9 @@ prometheus:
     enableFeatures:
       - memory-snapshot-on-shutdown
       - new-service-discovery-manager
-    image: 
-      repository: quay.io/prometheus/prometheus
+    image:
+      registry: quay.io
+      repository: prometheus/alertmanager
```

Comment:
Shouldnt we move it to UBI?

Reply:
Since it's telemetry-shippers, I'd prefer first try it internally for a while an later change on our public repo.